src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Imac"
+        class="card__image"
+      />
+
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <span>Product code: 195434</span>
+
+      <div class="rating">
+        <div class="rating__block">
+          <ul class="rating__list">
+            <li class="rating__stars"></li>
+            <li class="rating__stars"></li>
+            <li class="rating__stars"></li>
+            <li class="rating__stars"></li>
+            <li class="rating__stars"></li>
+          </ul>
+          <span>Reviews: 5</span>
```

Comment:
The reviews count should also be wrapped in a tag, such as <span> or <p>, to separate it from the static text for styling and semantic purposes.